packs/feats/archetype/bellflower-tiller/scarecrow.json

```
@@ -29,7 +29,18 @@
             "remaster": false,
             "title": "Pathfinder #147: Tomorrow Must Burn"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Scarecrow",
+                "selector": "weapon-damage",
+                "value": "@weapon.system.damage.dice",
+                "type": "circumstance",
+                "predicate": [
+                    "bellflower-scarecrow"
+                ]
+            }
+        ],
```

We can just omit the label since it's just the name of the feat, but keep in mind for the future that plain text needs to be localized. Your RE properties should be sorted in alphabetical order as well, which the extractor will do for you, among other things, so it's good practice to make changes within Foundry and then extract.
```suggestion
        "rules": [
            {
                "key": "FlatModifier",
                "predicate": [
                    "bellflower-scarecrow"
                ],
                "selector": "weapon-damage",
                "type": "circumstance",
                "value": "@weapon.system.damage.dice"
            }
        ],
```